staff/sergiomons/css-layout-simple-adaptive/styles/desktop.css

```diff
@@ -0,0 +1,34 @@
+* {
+    box-sizing: border-box;
+    padding: 0;
+    margin: 0;
+ }
+ html, body {
+    min-height: 100% !important;
+    height: 100%;
+    background-color: black;
+    color:white;
+    text-align:center;
+}
+header, footer{
+  width:100%;
+  height: 20%;
+  border: 1px solid white;
+}
+main {
+  width:100%;
+  height: 60%;
+  border: 1px solid white;
+}
+nav, aside{
+   width:20%;
+   float: left;
+   height: 100%;
+   border: 1px solid white;
+}
+section {
+    width: 60%;
+    float: left;
+    height: 100%;
```

Comment:
TIP group selectors for equal rules.

SEE Grouping Selectors https://www.w3schools.com/css/css_syntax.asp

Reply:
OK, I,m going to group them!